.travis.yml

```diff
@@ -31,6 +31,7 @@ matrix:
     - env: PRODUCT_TESTS_SUITE=suite-3
     - env: PRODUCT_TESTS_SUITE=suite-4
     - env: PRODUCT_TESTS_SUITE=suite-5
+    - env: PRODUCT_TESTS_SUITE=suite-6-non-generic
```

Comment:
I would remove `-non-generic` suffix. I think a comment in the file is much better. I think it might be hard to maintain that.